video_ingest_tool/cli.py

```
@@ -0,0 +1,857 @@
+                headers = results[0].keys()
+                for header in headers:
+                    table.add_column(header.replace("_", " ").title())
+                
+                for item in results:
+                    table.add_row(*[str(item.get(header, "N/A")) for header in headers])
+            
+            console.print(table)
+
+    except ValueError as e:
+        console.print(f"[red]Error: {e}[/red]")
+        raise typer.Exit(code=1)
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)